### ISSUE TITLE
Provide a basic simple example

### ISSUE THREAD

**Issue by rwb196884** [Bassman2/MediaDevices#127]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by NilsHoyer** [comment]
I agree with you: The documentation is not available for this project. So we have to use the source code to figure out, how it works. But I was able to do this as well, perhaps you can to it too?

**Comment by Bassman2** [comment]
In version 1.x, the documentation is included as a .chm file. 
From version 2.x onwards, the documentation can be found at https://bassman2.github.io/MediaDevices/.
Small code examples can be found in the tests at https://github.com/Bassman2/MediaDevices/tree/main/Test/MediaDevicesUnitTestShare.
